external/llvm-project/mlir/include/mlir/Dialect/Tosa/Transforms/Passes.td

```diff
@@ -52,7 +52,13 @@ def TosaPartition : Pass<"tosa-partition", "ModuleOp"> {
   let constructor = "createTosaPartitionPass()";
 
   let options = [
-    Option<"nofront", "nofront", "bool", /*default=*/"false",
+    ListOption<"primaryOps", "primary-ops", "std::string",
```

Comment:
See the test file for usage examples.  I could be persuaded to use "anchor" instead of "primary" but I'm sort-of waiting for something better.